backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -501,3 +504,21 @@ def create(self, validated_data: dict) -> ReviewResponse:
         process_review_response.delay(review_response.pk)
 
         return review_response
+
+
+class AuditTrailItemSerializer(serializers.ModelSerializer):
+    message = serializers.SerializerMethodField()
+    user = UserSerializer()
+
+    class Meta:
+        model = TimelineLog
+        fields = (
+            "pk",
+            "timestamp",
+            "user",
+            "message",
+            "extra_data",
+        )
+
+    def get_message(self, obj: TimelineLog) -> str:
+        return re.sub(r"\[.+\]: ", "", obj.get_message())
```

Comment:
Maybe let's update the template instead of changing the message on the in the serializer. I had used that format in the template just because it was what we had in the previous version!

Reply:
Ah ok, makes sense. Will update the template then 👍